src/mumble/AudioOutput.h

```diff
@@ -9,19 +9,14 @@
 #include <QtCore/QObject>
 #include <QtCore/QThread>
 #include <boost/shared_ptr.hpp>
+#include <memory>
```

Comment:
We usually keep includes separated by type. So Qt includes, Mumble includes, Boost includes and stdlib includes form their own blocks, separated by an empty line.